testing/darktable/APKBUILD

```diff
@@ -1,21 +1,24 @@
 # Contributor: Kevin Daudt <ops@ikke.info>
 # Maintainer: Kevin Daudt <ops@ikke.info>
-_tag=release-2.2.0rc0
+_ver=2.2.0
+_rc=rc1
+_tag=release-${_ver}${_rc}
 pkgname=darktable
-pkgver=2.2.0_rc0
+pkgver=${_ver}_${_rc}
```

Comment:
This breaks `abump`, i.e. complicates easy updates. You can convert pkgver to required format even without splitting it to two variables, e.g. `${pkgver/_/.}` to get `2.2.0.rc0` and `${pkgver/_/~}` to get `2.2.0~rc0`.